src/routers/mailingList/validator.ts

```diff
@@ -4,9 +4,8 @@ const validator = (route: string) => {
   switch (route) {
     case '/':
       return [
-        check('email').normalizeEmail().isEmail(),
+        check('email').isString(),
```

Comment:
This will allow the email field to be `undefined` or falsy (the empty string in our case) but still normalize and validate if a value is provided. We should probably also update the `MailingListRequestBody` type to reflect the optional email field.

```suggestion
        check('email')
          .optional({ checkFalsy: true })
          .normalizeEmail()
          .isEmail(),
```

Reply:
I'm going to leave the type because email will either be empty string or a valid email